code/game/gamemodes/clock_cult/clock_structures/mania_motor.dm

```diff
@@ -85,12 +84,12 @@
 			if(distance <= 1)
 				if(!H.Adjacent(src))
 					H << "<span class='sevtug'>\"[text2ratvar(pick(close_messages))]\"</span>"
-					H.playsound_local(T, hum, sound_distance, 1)
+					H.playsound_local(T, 'sound/effects/screech.ogg', sound_distance, 1)
```

Comment:
I think you should keep the variable, just make it a string, "sound/effects/screech/ogg" rather than the `get_sfx(...)` so it's easier to manage the sound